invenio_records_resources/services/records/schema.py

```diff
@@ -36,6 +36,7 @@ def clean(self, data, **kwargs):
         Why: We want to allow the output of a Schema dump, to be a valid input
              to a Schema load without causing strange issues.
         """
+        data = deepcopy(data)
```

Comment:
minor: maybe a rename 
```suggestion
        data = deepcopy(original_data)
```

Reply:
`original_data` might not be the best since we can do `pass_original` on `pre_load`. Changed it to input_data.